src/ident/Ident.h

```diff
@@ -28,12 +28,6 @@ namespace Ident
  */
```

Comment:
```
 * will call Ident::Init() itself if the manager has not been initialized already.
```

I cannot attach a change request to it, but please delete the stale line 27 above. Besides referring to the function removed by this PR, that line has become an irrelevant implementation detail.

Please do not forget to replace `,` on line 26 with `.` afterwords.

Reply:
Done in [974d953](https://github.com/squid-cache/squid/pull/991/commits/974d953b779f5ba1148d63484fd7ef442ea46f60)